inspirehep/modules/forms/static/scss/forms/form.scss

```diff
@@ -31,56 +31,66 @@
 }
 
 .ui-autocomplete-loading {
-     background: white url('/static/images/loading.gif') right top no-repeat;
+  background: white url('/static/images/loading.gif') right top no-repeat;
```

Comment:
Why the switch to 2 spaces for indentation? The rest of the CSS codebase seems to use 4...